content/general/limits.textile

```diff
@@ -75,14 +75,14 @@ The quotas for an account can be considered a soft limit, with no restrictions c
 
 |_/2=. Limit |_\2=. Free |_\2=. Self-service |_\2=. Business (legacy) |_\2=. Enterprise |
 |_. Soft |_. Hard |_. Soft |_. Hard |_. Soft |_. Hard |_. Soft |_. Hard |
-| "Peak connections":#peak-connections | 100 | 120 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
-| "Peak channels":#peak-channels | 100 | 120 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
-| "Total messages":#messages | 3,000,000 | 3,600,000 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
-| "Messages":#messages (per hour) | 41,700 | 50,000 | Quota / 72 | 2.5x quota / 72 | Quota / 72 | 5x quota / 72 |\2. Custom |
-| "Message rate":#messages (per second) | - | 35 | - | Hourly * 2.5 | - | Hourly * 2.5 |\2. Custom |
-| "Total bandwidth":#bandwidth (GiB) | 5.7 | 6.9 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
-| "Bandwidth":#bandwidth (per hour in MiB) | 81.38 | 97.66 | Quota / 72 | 2.5x quota / 72 | Quota / 72 | 5x quota / 72 |\2. Custom |
-| "Bandwidth rate":#bandwidth (per second in KiB ) | - | 69 | - | Hourly * 2.5 | - | Hourly * 2.5 |\2. Custom |
+| "Peak connections":#peak-connections | 200 | 240 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
+| "Peak channels":#peak-channels | 200 | 240 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
+| "Total messages":#messages | 6,000,000 | 7,200,000 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
+| "Messages":#messages (per hour) | 84,000 | 100,000 | Quota / 72 | 2.5x quota / 72 | Quota / 72 | 5x quota / 72 |\2. Custom |
+| "Message rate":#messages (per second) | - | 70 | - | Hourly * 2.5 | - | Hourly * 2.5 |\2. Custom |
+| "Total bandwidth":#bandwidth (GiB) | 11.5 | 13.8 | Quota | 2.5x quota | Quota | 5x quota |\2. Custom |
+| "Bandwidth":#bandwidth (per hour in MiB) | 160 | 190 | Quota / 72 | 2.5x quota / 72 | Quota / 72 | 5x quota / 72 |\2. Custom |
+| "Bandwidth rate":#bandwidth (per second in KiB ) | - | 110 | - | Hourly * 2.5 | - | Hourly * 2.5 |\2. Custom |
```

Comment:
These 3 rows are in GiB, MiB and KiB respectively, but the values that have been added are as GB, MB and KB.